src/unistd.rs

```diff
@@ -819,6 +821,441 @@ pub fn mkstemp<P: ?Sized + NixPath>(template: &P) -> Result<(RawFd, PathBuf)> {
+#[cfg(any(target_os="dragonfly", target_os="freebsd", target_os = "ios",
+          target_os="linux", target_os = "macos", target_os="openbsd"))]
+_POSIX_TYPED_MEMORY_OBJECTS = libc::_SC_TYPED_MEMORY_OBJECTS,
+_POSIX_VERSION = libc::_SC_VERSION,
+#[cfg(any(target_os="dragonfly", target_os="freebsd", target_os = "ios",
+          target_os="linux", target_os = "macos", target_os="netbsd",
+          target_os="openbsd"))]
+_POSIX_V6_ILP32_OFF32 = libc::_SC_V6_ILP32_OFF32,
+#[cfg(any(target_os="dragonfly", target_os="freebsd", target_os = "ios",
+          target_os="linux", target_os = "macos", target_os="netbsd",
+          target_os="openbsd"))]
+_POSIX_V6_ILP32_OFFBIG = libc::_SC_V6_ILP32_OFFBIG,
+#[cfg(any(target_os="dragonfly", target_os="freebsd", target_os = "ios",
+          target_os="linux", target_os = "macos", target_os="netbsd",
+          target_os="openbsd"))]
+_POSIX_V6_LP64_OFF64 = libc::_SC_V6_LP64_OFF64,
+#[cfg(any(target_os="dragonfly", target_os="freebsd", target_os = "ios",
+          target_os="linux", target_os = "macos", target_os="netbsd",
+          target_os="openbsd"))]
+_POSIX_V6_LPBIG_OFFBIG = libc::_SC_V6_LPBIG_OFFBIG,
+_POSIX2_C_BIND = libc::_SC_2_C_BIND,
+_POSIX2_C_DEV = libc::_SC_2_C_DEV,
+_POSIX2_CHAR_TERM = libc::_SC_2_CHAR_TERM,
+_POSIX2_FORT_DEV = libc::_SC_2_FORT_DEV,
+_POSIX2_FORT_RUN = libc::_SC_2_FORT_RUN,
+_POSIX2_LOCALEDEF = libc::_SC_2_LOCALEDEF,
+#[cfg(any(target_os="dragonfly", target_os="freebsd", target_os = "ios",
+          target_os="linux", target_os = "macos", target_os="netbsd",
+          target_os="openbsd"))]
+_POSIX2_PBS = libc::_SC_2_PBS,
+#[cfg(any(target_os="dragonfly", target_os="freebsd", target_os = "ios",
+          target_os="linux", target_os = "macos", target_os="netbsd",
+          target_os="openbsd"))]
+_POSIX2_PBS_ACCOUNTING = libc::_SC_2_PBS_ACCOUNTING,
+#[cfg(any(target_os="dragonfly", target_os="freebsd", target_os = "ios",
+          target_os="linux", target_os = "macos", target_os="netbsd",
+          target_os="openbsd"))]
+_POSIX2_PBS_CHECKPOINT = libc::_SC_2_PBS_CHECKPOINT,
+#[cfg(any(target_os="dragonfly", target_os="freebsd", target_os = "ios",
+          target_os="linux", target_os = "macos", target_os="netbsd",
+          target_os="openbsd"))]
+_POSIX2_PBS_LOCATE = libc::_SC_2_PBS_LOCATE,
+#[cfg(any(target_os="dragonfly", target_os="freebsd", target_os = "ios",
+          target_os="linux", target_os = "macos", target_os="netbsd",
+          target_os="openbsd"))]
+_POSIX2_PBS_MESSAGE = libc::_SC_2_PBS_MESSAGE,
+#[cfg(any(target_os="dragonfly", target_os="freebsd", target_os = "ios",
+          target_os="linux", target_os = "macos", target_os="netbsd",
+          target_os="openbsd"))]
+_POSIX2_PBS_TRACK = libc::_SC_2_PBS_TRACK,
+_POSIX2_SW_DEV = libc::_SC_2_SW_DEV,
+_POSIX2_UPE = libc::_SC_2_UPE,
+_POSIX2_VERSION = libc::_SC_2_VERSION,
+PAGE_SIZE = libc::_SC_PAGE_SIZE,
+// POSIX requires an alias PAGESIZE == PAGE_SIZE, but Rust doesn't like
```

Comment:
It'd be nice if this were able to be documented in the docs for this enum value. Maybe add a doc comment for `PAGE_SIZE` that says this as well since that way the user could search `nix` docs and find this instead of having to search the source code.